style.css

```diff
@@ -0,0 +1,92 @@
+html {
+  font-size: 10px;
+}
+body {
+  font-size: 1.6rem;
+}
+.container {
+  max-width: 120rem;
+  margin: 0 auto;
+}
+.json-section {
+  margin: 0 auto;
+  width: 50%;
+  display: flex;
+  justify-content: center;
+  flex-direction: column;
+}
+.json-section__title {
+  margin: 2rem auto;
+  height: 5rem;
+  width: 50%;
+  display: flex;
+  flex-direction: row;
+}
+.json-section__title--text {
+  display: flex;
+  align-items: center;
+  border-bottom: 0.1rem solid #afafaf;
+  width: 25rem;
+  margin-left: 5rem;
+  text-transform: capitalize;
+}
+.json-section__title--label {
+  display: flex;
+  align-items: center;
+}
+.json-section__title--label input {
+  position: absolute;
+  opacity: 0;
+  cursor: pointer;
+}
+
+.json-section__title--label .check-custom {
+  position: absolute;
+  height: 2.4rem;
+  width: 2.4rem;
+  background-color: transparent;
+  border-radius: 0.5rem;
+  border: 0.1rem solid #8b8b8b;
+  cursor: pointer;
+}
+
+.json-section__title--label .check-custom::after {
+  position: absolute;
+  content: "";
+  left: 1.2rem;
+  top: 1.2rem;
+  height: 0;
+  width: 0;
+  border-radius: 0.5rem;
+  border: solid #8b8b8b;
+  border-width: 0 0.1rem 0.1rem 0;
+  -webkit-transform: rotate(0deg) scale(0);
+  -ms-transform: rotate(0deg) scale(0);
+  transform: rotate(0deg) scale(0);
+  opacity: 1;
+}
+
+.json-section__title--label input:checked ~ .check-custom {
+  background-color: #fd3d6c;
+  border-radius: 0.5rem;
+  -webkit-transform: rotate(0deg) scale(1);
+  -ms-transform: rotate(0deg) scale(1);
+  transform: rotate(0deg) scale(1);
+  opacity: 1;
+  border: 2px solid transparent;
+}
+
+.json-section__title--label input:checked ~ .check-custom::after {
```

Comment:
What does `~` mean?

Reply:
The tilde (~) separates the two selectors